src/GitHub.Api/Git/Tasks/GitLfsVersionTask.cs

```diff
@@ -0,0 +1,19 @@
+using System;
+using System.Threading;
+
+namespace GitHub.Unity
+{
+    class GitLfsVersionTask : ProcessTask<Version>
+    {
+        private const string TaskName = "git lfs version";
+
+        public GitLfsVersionTask(CancellationToken token, IOutputProcessor<Version> processor = null)
+            : base(token, processor ?? new LfsVersionOutputProcessor())
+        {
+            Name = TaskName;
+        }
+
+        public override string ProcessArguments { get { return "lfs version"; } }
+        public override TaskAffinity Affinity { get { return TaskAffinity.Concurrent; } }
```

Comment:
This is the default concurrency, just fyi (being explicit about it is fine)

Reply:
Cool